codeforamerica/vita-min
# FYST-1221 Update content on return status page for AZ with some global updates

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1221
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- this story was originally meant to make sure the page displays the correct status but it seems to already be doing that so i just updated some of the content since i'd already been working on FYST-1065
## How to test?
- so far i've only "tested" it by looking at the page locally with Anu, but open to writing some simple controller specs to check the content. that's probably a good idea actually
## Screenshots (for visual changes)
- Before
  ![Screenshot 2024-12-03 at 2 29 00 PM](https://github.com/user-attachments/assets/8044346c-353d-493f-bc45-a307086def30)
- After

accepted
refund:
![Screenshot 2024-12-03 at 11 31 28 AM](https://github.com/user-attachments/assets/aad02c20-2aba-4e92-9a0d-9fae10c7a69c)
owed:
![Screenshot 2024-12-03 at 2 24 07 PM](https://github.com/user-attachments/assets/ff35383f-b1d8-46ff-b5bf-6a68e9070550)

pending
![Screenshot 2024-12-03 at 3 07 47 PM](https://github.com/user-attachments/assets/1d648a40-e397-4226-8e3d-598328b82515)
